index.html

```diff
@@ -0,0 +1,233 @@
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <ul>
+                                <li class="project-language"> Html<li>
+                                <li class="project-language"> Bootstrap<li>
+                                <li class="project-language"> Ruby<li>
+                            </ul>
+                        </div>
+                        <a href="#" class="project-btn">See project</a>
+                        
+                     </div> 
+                     <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <ul>
+                                <li class="project-language"> Html<li>
+                                <li class="project-language"> Bootstrap<li>
+                                <li class="project-language"> Ruby<li>
+                            </ul>
+                        </div>
+                        <a href="#" class="project-btn">See project</a>
+                        
+                     </div>
+
+                  
+                 </div>
+                </section>
+
+            <section class="about">
+                <div class="description">
+                    <h2>About me</h2>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                    <a href="#">Get My Resume</a>
+                </div>
+               <div class="list">
+                   
+                   <h3><img src="./images/rectanglebtn.png" class="rectangle-icon" alt="rectangle-icon">Language</h3>
+                   <ul>
+                        <li>Javascript</li>
+                       <li>Ruby</li>
+                       <li>Html</li>
+                       <li>Css</li>
+                   </ul>
+
+                   <h3><img src="./images/squareicon.png" class="square-icon" alt="square-icon">Frameworks</h3>
+                   <ul>
+                    <li>bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                   </ul>
+
+               <h3><img src="./images/circleicon.png" class="square-icon" alt="skills-icon">Skills</h3>
```

Comment:
- [ ] Your 'Languages', 'Frameworks' and 'Skills' categories should be inside a `ul`, with each of them in an `li`

![langs](https://user-images.githubusercontent.com/36542986/134501932-41a4c17a-7b38-4634-9ec1-26580b7517b0.jpg)

Reply:
Thanks for taking your time to review my project.Am going to work on the requested changes